idunn/blocks/stars.py

```diff
@@ -20,6 +20,13 @@ class StarsKind(Enum):
     LODGING = "lodging"
     RESTAURANT = "restaurant"
 
+    def check_class(self, subclass: Optional[str]) -> bool:
+        return (self, subclass) in [
+            (self.LODGING, "lodging"),
+            (self.RESTAURANT, "restaurant"),
+            (self.RESTAURANT, "fast_food"),
+        ]
```

Comment:
`subclass` should rather be named `class_name`

Reply:
I just rewrote last commit as this is a very minor change.